frontend/serve/ssr/page_test.go

```diff
@@ -0,0 +1,50 @@
+package ssr
+
+import (
+	"testing"
+
+	"github.com/short-d/app/fw/assert"
+)
+
+func TestRenderPage(t *testing.T) {
+	testPage := `
+jkas'x'n'j'kkj
+{{SSR_OG_TITLE}}
+nnm'zxc;nkkas
+{{SSR_OG_DESCRIPTION}}
+pqwepc['xzmkm
+{{SSR_OG_IMAGE}}
+llasdm21k.asda2
+{{SSR_TWITTER_TITLE}}
+023elksdlmcs;
+{{SSR_TWITTER_DESCRIPTION}}
+lzxckmzcxmkm23
+{{SSR_TWITTER_IMAGE}}
+[;sd-p32er2l,csdc
+`
+	ssrVars := map[string]string{
+		"OG_TITLE":            "Title",
```

Comment:
```suggestion
		"OG_TITLE":            "OpenGraphTitle",
```

Reply:
We need to confident make sure the content is rendered at the correct location by distinguishing `OpenGraph Title` from `Twitter Title`.